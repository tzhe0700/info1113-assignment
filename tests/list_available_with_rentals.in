ADD STUDENT Harry Potter
ADD COLLECTION spellbooks.csv
RENT 100000 111111
LIST AVAILABLE
EXIT
