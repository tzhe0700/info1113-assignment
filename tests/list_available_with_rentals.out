user: Success.

user: 1 spellbooks successfully added.

user: Success.

user: No spellbooks available.

user: Ending Archive process.
